src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -226,5 +248,29 @@ private void SortableColumnHeader_PreviewKeyUp(object sender, KeyEventArgs e)
                 e.Handled = true;
             }
         }
+
+        private GridViewColumn CreateRequestedVersionColumn()
```

Comment:
I find this weird, but sounds like it solves the narrator problem. 
I'd wait on a review from someone skilled with xaml stuff.

Reply:
@rrelyea & @srdjanjovcic